packages/javascript/bh-shared-ui/src/hooks/useTagLimits/useTagLimits.ts

```diff
@@ -38,7 +38,7 @@ export const useTagLimits = () => {
     }
 
     const zonesCount = tagsQuery.data.reduce(zoneReducer, 0);
-    const labelsCount = tagsQuery.data.length - zonesCount;
+    const labelsCount = tagsQuery.data.filter((label) => label.type === 2).length;
```

Comment:
I wonder if it'd help simplify if we swapped to a `tagsQuery.Data?.forEach` approach and incrementing both zone and label count instead of these 2 different approaches?